File: c++/1180.cpp
#include <iostream>

using namespace std;

int main() {

    int n, a, p, menor;
    cin >> n;
    int x[n];

    for(int i = 0; i < n; i++) {
        cin >> a;
        x[i] = a;
        if(i == 0) {
            menor = x[0];
        }
        if(x[i] < menor) {
            menor = x[i];
            p = i;
        }
    }
    cout << "Menor valor: " << menor << endl;
    cout << "Posicao: " << p << endl;

    return 0;
}

File: python3/1180.py
tam = int(input())
entrada = input().split()
entrada = [int(i) for i in entrada]

menor = entrada[0]
pos = 0

for index in range(tam):
    if(entrada[index] < menor):
        menor = entrada[index]
        pos = index

print("Menor valor: {}".format(menor))
print("Posicao: {}".format(pos))
